ads/_config.js

```diff
@@ -727,6 +727,10 @@ export const adConfig = {
 
   'postquare': {},
 
+  'pressboard': {
+    renderStartImplemented: true,
```

Comment:
I couldn't see the `render-start` signal when I test locally, could you double check that the renderStart API is called correctly? Thank you.